### PR TITLE
Docs: feat: make automatic system locking for cluster node (CaaSP) user configurable

### PR DESCRIPTION
**Pull request by mbologna**
This is the accompanying doc of PR https://github.com/uyuni-project/uyuni/pull/2187

This is targeting Manager-4.1 and Uyuni.

### CODE REVIEW

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 1]
```diff
@@ -24,13 +24,30 @@ When a system is locked, the {webui} shows a warning and you can schedule action
 ====
 The locking mechanism works only with plain Salt minions (locking is not supported with salt-ssh minions).
 
-You can enable or disable the system lock using the System Lock formula.
+You can enable or disable the system lock by using Formulas > System Lock > uncheck "Lock system" and press "Save Formula" (applying the highstate is not needed for this formula).
```
Comment (Loquacity) [inline modules/client-configuration/pages/vhm-caasp.adoc:27]:
```suggestion
You can enable or disable the system lock by navigating to menu:Formulas[System Lock].
Uncheck [guimenu]``Lock system`` and click btn:[Save Formula] to save.
You do not need to apply the highstate for this formula.
```

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 2]
```diff
@@ -24,13 +24,30 @@ When a system is locked, the {webui} shows a warning and you can schedule action
...
+java.automatic_system_lock_cluster_nodes = false
+----
+
+. Restart the spacewalk service to pick up the changes:
++
+----
+spacewalk-service restart
+----
+
 ====
```
Comment (Loquacity) [inline modules/client-configuration/pages/vhm-caasp.adoc:51]:
```suggestion
The System Lock formula is enabled automatically if {caasp} is detected on the node, or if you perform an action that involves a package.
This does not include actions that involve patches.

You can disable the System Lock formula from being automatically applied by editing the configuration file.
Open [path]``/etc/rhn/rhn.conf`` and add this line at the end of the file:

----
java.automatic_system_lock_cluster_nodes = false
----

Restart the spacewalk service to pick up the changes:

----
spacewalk-service restart
----
```

**Comment by mbologna** [comment]
@Loquacity waiting for your OK after having addressed your issues

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 3]
```diff
@@ -24,32 +24,28 @@ When a system is locked, the {webui} shows a warning and you can schedule action
...
-====
-The System Lock formula is enabled automatically if {caasp} is detected on the node.
-Additionally, any action involving packages (not patches) triggers the automatic apply
-of the System Lock formula.
+The System Lock formula is enabled automatically if {caasp} is detected on the node, or if you perform an action that involves a package.
+This does not include actions that involve patches.
 
-If you want to disable the automatic apply of the System Lock formula:
+You can disable the System Lock formula from being automatically applied by editing the configuration file.
+Open [path]``/etc/rhn/rhn.conf`` and add this line at the end of the file:
```
Comment (keichwa) [inline modules/client-configuration/pages/vhm-caasp.adoc:37]: I'd just write:
```
Add this line at the end of the [path]``/etc/rhn/rhn.conf`` file:

Comment (mbologna) [reply modules/client-configuration/pages/vhm-caasp.adoc:37]: Done, thanks for the hint!

**Comment by mbologna** [comment]
Can we please make this doc change available in Manager-4.0 too?

**Comment by Loquacity** [comment]
> Can we please make this doc change available for the next MU of Manager-4.0?

Yep, it's merged, so it will be in :+1:

**Pull request merged**